### ISSUE TITLE
Add arrow controls on checkbox/radio

### ISSUE THREAD

**Issue by VanishMax** [VanishMax/industrial-ui-vue#1]
To improve accessibility, checkbox and radio components that are in the group should be controlled by arrow keys. 

Checkboxes are checked by pressing enter/space
